src/Node/YieldReady.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Node;
+
+/**
+ * Marks nodes that are ready for using "yield" instead of "echo" or "print()" for rendering.
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class YieldReady
```

Comment:
should this class really be in the `Node` namespace ? This namespace only contains nodes until now AFAICT.

Reply:
It fits there to me. Of course let me know if you have a better idea.

Reply:
I suggest putting it in the `Twig\Attribute` namespace instead, which will also be used by https://github.com/twigphp/Twig/pull/3916